src/common/base_client.ts

```diff
@@ -358,7 +376,9 @@ export abstract class BaseClient {
         this.consumerRpcs.set(KolibriRequestMethods.UserNotifyRequestMethod,
             this.handleUserNotifyRequest.bind(this));
         this.consumerRpcs.set(KolibriRequestMethods.CommitRequestMethod,
-            this.handleCommitRequest.bind(this));
+            this.handleCommitRequest.bind(this));   
```

Comment:
remove trailing space after handleCommitRequest.bind(this));

Reply:
good catch, done